src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb

```diff
@@ -14,12 +14,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Sub
 
         Friend Overrides Function FormatSourceSpan(span As LinePositionSpan, formatter As IFormatProvider) As String
-            Return String.Format("({0}) ", span.Start.Line + 1)
+            Return "(" & span.Start.Line + 1 &") "
```

Comment:
>span.Start.Line + 1 [](start = 25, length = 19)

Should probably call ToString explicitly to avoid an invocation of VB runtime helper.